Logger/BaseLoggerMixins.cs

```diff
@@ -1,7 +1,59 @@
-﻿namespace Logger
+﻿using System;
+
+namespace Logger
 {
     public static class BaseLoggerMixins
     {
-        
+        public static void Error(this BaseLogger baseLogger, string message, params object[] args)
+        {
+            if(baseLogger is null)
+            {
+                throw new ArgumentNullException("baseLogger is null");
+            }
+            else
+            {
+                LogLevel logLevel = LogLevel.Error;
+                baseLogger.Log(logLevel, String.Format(message, args));
```

Comment:
From the coding guidelines

> DO use the C# keyword rather than the BCL name when specifying a data type (e.g., string rather than String).

```suggestion
                baseLogger.Log(logLevel, string.Format(message, args));
```